README.md

```diff
@@ -25,13 +25,13 @@ For building the Nighthawk test server, see [here](source/server/README.md).
 ### Test it
 
 ```bash
-bazel test -c dbg //test:nighthawk_test
+bazel test -c dbg //test/... //integration/...
 ```
 
 ### Build it
 
 ```bash
-bazel build -c opt //:nighthawk_client
+bazel build -c opt //:nighthawk_client nighthawk_test_server nighthawk_service
```

Comment:
Does this work? Don't the latest binaries need a path? Also, should we just have a filegroup or something for this?

Reply:
Yes this works. I will look into creating a filegroup.